### PR TITLE
Tweaks to Collapsing of Menus

### PR DESCRIPTION
**Pull request by lolfuljames**
Add Quality of Life Improvement for Mobile

-Autosearch when user gps is available
-Bottom menu closes when top is expanded, vice-versa
-Fixes the bug whereby no restaurant found wasn't being shown.
-Replaces hiding result container with collapsing of result container
 upon pressing search.
-Change order of showing information (delaying it to final parts so information is only shown when ready, photos of restaurants are an exception to this)

demo: https://jtan-sps-summer20.appspot.com/

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -136,24 +136,24 @@ function updateRestaurant(result) {
...
-    }
-
     // Hide next and previous buttons if only one restaurant
     if(result.length === 1) {
       document.getElementById("restaurant-buttons").style.display = "none";
     } else {
       document.getElementById("restaurant-buttons").style.display = "flex";
     }
+    resContainerElement.classList.remove("hide");
+    resContainerElement.scrollTop = 0;
```
Comment (jieliangang) [inline src/main/webapp/script.js:148]: Can I check what is scrollTop = 0 for?

**Comment by jieliangang** [comment]
When a new restaurant is searched, the bottom tab shrinks and then expands back when the result is retrieved. I'm not sure whether this effect is better than having it remained shown.

File: src/main/webapp/script.js (hunk 1 above)
Comment (lolfuljames) [reply src/main/webapp/script.js:148]: Scrolls back to top of the element, applicable to mobile when element is larger than max height

**Review by jieliangang — APPROVED** [review]
LGTM

**Pull request merged**